Dockerfile

```diff
@@ -16,9 +15,14 @@ ENV PATH="$PATH:$NPM_PACKAGES/bin"
 ENV MANPATH="$MANPATH:$NPM_PACKAGES/share/man"
 
 # Setup a global packages location for "node" user so we can npm link
-RUN mkdir $NPM_PACKAGES \
-    && npm config set prefix $NPM_PACKAGES
+RUN mkdir $NPM_PACKAGES && chmod -R 777 $NPM_PACKAGES
+  
+RUN npm config set prefix $NPM_PACKAGES && npm install -g npm@9.2.0
```

Comment:
I'm not a big fan of hard coding a version of `npm` here and as the other packages (`typescript`, `electron`, etc.) at certain versions later. I get needing rimraf, but is the rest necessary?  Shouldn't it be pulling these from the package.json?